frontend/src/store/FamLoginUserState.ts

```diff
@@ -113,6 +113,18 @@ const hasAccessRole = (role: string): boolean => {
     return false;
 };
 
+const hasAccess = (role: any): boolean => {
```

Comment:
So if we set `requiredPrivileges: [APP_ADMIN_ROLE]` above, and no need need method anymore

Reply:
Yeah, it isn't needed in this case. Nice catch.